src/main/java/propertyFiles/Bundle.properties

```diff
@@ -235,6 +235,37 @@ notification.mail.import.filesystem=Dataset {2} ({0}/dataset.xhtml?persistentId=
 notification.import.filesystem=Dataset <a href="/dataset.xhtml?persistentId={0}" title="{1}">{1}</a> has been successfully uploaded and verified.
 notification.import.checksum=<a href="/dataset.xhtml?persistentId={0}" title="{1}">{1}</a>, dataset had file checksums added via a batch job.
 removeNotification=Remove Notification
+
+# These are the labels of the options where the muted notifications can be selected by the users
+notification.muteOptions=Notification and mailing settings
```

Comment:
```suggestion
notification.muteOptions=Notification settings
```

I'm not in love with "Notification and mailing settings". (I'll put screenshots below for context.) "Notification settings" is shorter and can encompass both email and in-app varieties.

![Screen Shot 2022-05-16 at 4 33 20 PM](https://user-images.githubusercontent.com/21006/168677900-39e20c77-9689-49dc-9010-f445c9c3bd14.png)


![Screen Shot 2022-05-16 at 4 25 39 PM](https://user-images.githubusercontent.com/21006/168676774-64cb0c08-85b1-4206-90a5-1ca00f3f85b3.png)